app/routes/project/tasks.js

```diff
@@ -2,9 +2,4 @@ import Ember from 'ember';
 
 const { Route } = Ember;
 
-export default Route.extend({
-  model() {
-    let project = this.modelFor('project');
-    return project;
-  }
-});
+export default Route.extend({});
```

Comment:
The hook is unnecessary. All subroutes that need a project record take it from `this.modelFor('project')` so this one is just a promise that resolves basically for nothing.